packages/js/src/index.test.ts

```diff
@@ -0,0 +1,27 @@
+import * as core from '.';
```

Comment:
what's the point of this file?

Reply:
test exports. not necessary enough.
Removed. (See [discussion](https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1645772028319539))